bin/bundle_ruby

```diff
@@ -41,6 +41,8 @@ module Bundler
   end
 end
 
+puts "Warning: bundle_ruby will be deprecated in Bundler 2.0.0."
```

Comment:
pretty sure this needs to go to STDERR, otherwise piping the output of `bundle_ruby` will be broken